### PR TITLE
Update taxonomy

### PR DESCRIPTION
**Pull request by hollensteiner**
Details here: https://contentful.tpondemand.com/entity/9866

### CODE REVIEW

File: apiary/_partials/cma/editing-interface.apib [hunk 1]
```diff
@@ -39,36 +39,36 @@ The Editing Interface would look like this:
...
 
-Please note that there are specific sets of applicable Widgets per Content Type field type:
+Please note that there are specific sets of applicable widgets per content type field type:
 
 | Widget ID               | Applicable field types        | Description               |
 |-------------------------|-------------------------------|---------------------------|
 | assetLinkEditor         | Asset                         | Search, attach, and preview an asset. |
-| assetLinksEditor        | Asset (Array)                 | Search, attach, reorder, and preview multiple assets. |
-| assetGalleryEditor      | Asset (Array)                 | Search, attach, reorder, and preview multiple assets in a gallery layout |
+| assetLinksEditor        | Asset (array)                 | Search, attach, reorder, and preview multiple assets. |
```
Comment (benben) [inline apiary/_partials/cma/editing-interface.apib]: I think data types like Array, Symbol, ... should stay capitalized

File: docs/android/tutorials/advanced-types.md [hunk 2]
```diff
@@ -167,16 +167,16 @@ And this will be the result in only one asset (again using the `cfexampleapi` sp
...
-In order to request all [ContentType](http://contentful.github.io/contentful.java/index.html?com/contentful/java/cda/CDAContentType.html)s,
+In order to request all [content types](http://contentful.github.io/contentful.java/index.html?com/contentful/java/cda/CDAContentType.html),
 you could use a call like this:
 
 ~~~ java
 CDAArray array = client.fetch(CDAContentType.class).all();
 ~~~
 
-Taking the resulting object, it'll be printed as (This example will only contain one ContentType, since the space only
+Taking the resulting object, it'll be printed as (This example will only contain one Content type, since the space only
```
Comment (realityking) [inline docs/android/tutorials/advanced-types.md]: Content should not be capitalised.

File: docs/ios/index.md [hunk 3]
```diff
@@ -43,18 +43,18 @@ Webinar showing how to use our CDA and CMA SDKs.<br>
...
 Tool for recreating your content model inside Xcode.<br>
 [View the tutorial](/developers/docs/ios/tutorials/using-contentful-xcode-plugin/)
 
-## Tools and framework integrations
+## Tools and integrations
 Here are some tools developed by our team to help you completing some basic tasks.
 
 ### Xcode plugin
-A plugin for automatically creating Core Data models from your Contentful content model. Read this to get up and running with it.<br>
+A plugin for automatically creating core data models from your Contentful content model. Read this to get up and running with it.<br>
```
Comment (neonichu) [inline docs/ios/index.md]: This should still be "Core Data" as it is a name

File: docs/php/tutorials/getting-started-with-contentful-and-php.md [hunk 4]
```diff
@@ -38,15 +38,15 @@ $client = new \Contentful\Delivery\Client('b4c0n73n7fu1', 'cfexampleapi');
...
 
-In Contentful, we separate content between Entries, which contain your data and relationships with other
-content or images, and Assets, which represent static content, like images, and are served as files. You can read more
-about that in our [Data Model Concepts guide][3].
+In Contentful, we separate content between entries, which contain your data and relationships with other
+content or images, and assets, which represent static content, like images, and are served as files. You can read more
+about that in our [data model concepts guide][3].
 
-In this section we'll address Entries. Assets will be addressed in a later section.
+In this section we'll address entries. assets will be addressed in a later section.
```
Comment (realityking) [inline docs/php/tutorials/getting-started-with-contentful-and-php.md]: Beginning of a sentence (Assets) should still be capitalised.

File: docs/ruby/index.md [hunk 5]
```diff
@@ -36,34 +36,34 @@ This tutorial will walk you through your first steps in using Contentful within
...
-## Unofficial Tools
+## Unofficial tools
 
 These third party libraries are **not officially supported** by Contentful and may be incomplete. If you want to include your own libraries on this list open a pull request that matches our [contribution guidelines](https://github.com/contentful-labs/awesome-contentful/blob/master/CONTRIBUTING.md).<br>
 
 ### Rails integration
 A collection of useful things to help make it easier to integrate Contentful into your Rails app.<br>
-[View on GitHub - Contentful Rails](https://github.com/contentful/contentful_rails)<br>
-[View on GitHub - Contentful Model](https://github.com/contentful/contentful_model)<br>
+[View on GitHub - Contentful rails](https://github.com/contentful/contentful_rails)<br>
```
Comment (neonichu) [inline docs/ruby/index.md]: "Rails" should stay that way as it is a name.

File: docs/php/tutorials/getting-started-with-contentful-and-symfony.md [hunk 6]
```diff
@@ -43,7 +43,7 @@ public function registerBundles()
 
 ## Configuration
 
-To configure the Bundle add the following section to your application's `config.yml`:
+To configure the bundle add the following section to your application's `config.yml`:
```
Comment (realityking) [inline docs/php/tutorials/getting-started-with-contentful-and-symfony.md]: Bundle is a Symfony specific term. They always capitalise it so I think so should we.

File: docs/php/tutorials/getting-started-with-contentful-and-symfony.md [hunk 7]
```diff
@@ -142,32 +142,33 @@ class DefaultController extends Controller
...
 
 To discover how to use the Contentful client, check out the
-[Getting Started with Contentful and PHP](/developers/docs/php/tutorials/getting-started-with-contentful-and-php/) tutorial.
+[getting started with Contentful and PHP](/developers/docs/php/tutorials/getting-started-with-contentful-and-php/) tutorial.
 
-## Using the Web Debug Toolbar
+## Using the web debug toolbar
 
-The ContentfulBundle integrates with Symfony's Web Debug Toolbar. If the toolbar is shown and there were requests to the
+The ContentfulBundle integrates with Symfony's web debug toolbar. If the toolbar is shown and there were requests to the
```
Comment (realityking) [inline docs/php/tutorials/getting-started-with-contentful-and-symfony.md]: Web Debug Toolbar is also their own term: http://symfony.com/blog/new-in-symfony-2-8-redesigned-web-debug-toolbar.

File: docs/javascript/index.md [hunk 8]
```diff
@@ -34,37 +34,36 @@ View in [JavaScript](/developers/api-demo/javascript/) |
...
 
 ### Angular-Contentful
-An AngularJS library to access the Contentful Content Delivery API<br>
+An AngularJS library to access the content delivery API<br>
 [View on GitHub](https://github.com/jvandemo/angular-contentful) - **This library is not officially supported**
 
 ## Apps
-Links to the source code of our in house built apps. Use them as examples when building your own product with Contentful.
-You can download them by looking for “Contentful” on Google Play.
+Links to the source code of our in house built apps. Use them as examples when building your own product with Contentful. You can download them by looking for “Contentful” on Google Play.
```
Comment (trodrigues) [inline docs/javascript/index.md]: > You can download them by looking for “Contentful” on Google Play.

I had never noticed this sentence before but this actually makes no sense to be here at all.

File: docs/php/tutorials/getting-started-with-contentful-and-symfony.md [hunk 9]
```diff
@@ -142,32 +142,33 @@ class DefaultController extends Controller
...
+on the the "Details" in the last column gives you an overview of the request and response and exceptions thrown by
 the Contentful SDK.
 
 {: .img}
 ![](https://images.contentful.com/256tjdsmm689/5jlwmBgQeWWMyuik2Mae6e/a01a4f1c897cfb759ff0b14f4b591bb4/symfony-web-profiler-details.png?w=800)
 
 ## Conclusion
 
-Now you should be familiar with the basics of how to use Contentful in a Symfony application. You can find the Bundle on
+Now you should be familiar with the basics of how to use Contentful in a Symfony application. You can find the bundle on
```
Comment (realityking) [inline docs/php/tutorials/getting-started-with-contentful-and-symfony.md]: Bundle again

**Comment by hollensteiner** [comment]
Thanks for the feedback, y'all - Fixed!
Additionally:
- After consulting w/ Eugene, decided to capitalize the APIs (Content Delivery API, Content Management API,...) & apps, tools, integrations we're publishing (Contentful Scheduler, Contentful Space Sync,...)
- Capitalized field types (Text, Symbol,...)

File: docs/ruby/index.md [hunk 10]
```diff
@@ -36,18 +36,18 @@ This tutorial will walk you through your first steps in using Contentful within
...
 To get you jump-started in Contentful using Contentful Bootstrap.<br>
 [View the tutorial](/developers/docs/ruby/tutorials/using-contentful-bootstrap-for-keeping-up-with-your-spaces/)
 
-## Tools and Framework Integrations
+## Tools and integrations
 Here are some tools developed by our team to help you completing some basic tasks.
 
-### Contentful Webhook Listener
-A Simple HTTP Webserver with pluggable behavior for listening to API Webhooks.<br>
+### contentful Webhook Listener
```
Comment (dlitvakb) [inline docs/ruby/index.md]: Why did you remove capitalization from `Contentful`? it's the first word of a sentence, also, I think Contentful should always be capitalized IIRC

File: apiary/_partials/cma/editing-interface.apib [hunk 11]
```diff
@@ -39,31 +39,31 @@ The Editing Interface would look like this:
...
 | datePicker              | Date                          | Select date, time, and timezon. |
 | entryLinkEditor         | Entry                         | Search and attach another entry.
-| entryLinksEditor        | Entry (Array)                 | Search and attach multiple entries. |
+| entryLinksEditor        | Entry (array)                 | Search and attach multiple entries. |
 | entryCardEditor         | Entry                         | Search, attach, and preview another entry. |
-| entryCardsEditor        | Entry (Array)                 | Search, attach, and preview multiple entries. |
-| numberEditor            | Integer, Number               | A simple input for numbers. |
-| rating                  | Integer, Number               | Uses stars to select a number. |
+| entryCardsEditor        | Entry (array)                 | Search, attach, and preview multiple entries. |
+| numberEditor            | Integer, number               | A simple input for numbers. |
```
Comment (realityking) [inline apiary/_partials/cma/editing-interface.apib]: Shouldn't number be capitalised too?

File: docs/tools/spacemanagement.md [hunk 12]
```diff
@@ -38,11 +38,10 @@ To get you jump-started in Contentful using Contentful Bootstrap. We also have a
 
 - [View on GitHub](https://github.com/contentful/contentful-bootstrap.rb)
```
Comment (dlitvakb) [inline docs/tools/spacemanagement.md]: It says we have a tutorial, but tutorial is not linked

File: apiary/_partials/content-types-description.md [hunk 13]
```diff
@@ -1,30 +1,30 @@
...
 items      |Schema        | Defines a subschema for the elements of an array field. This is required when `type` is `"Array"`.
-required   |Boolean       |Describes whether the Field is mandatory.
-localized  |Boolean       |Describes whether the Field will support different values for different locales.
-disabled   |Boolean       |Describes whether the Field is disabled. Disabled fields are hidden in the editing application.
-omitted    |Boolean       |If set to `true` fully omits this field in the Content Delivery API and Content Preview API responses. The Content Management API is not affected by this.
+required   |Boolean       |Describes whether the field is mandatory.
+localized  |Boolean       |Describes whether the field will support different values for different locales.
+disabled   |Boolean       |Describes whether the field is disabled. Disabled fields are hidden in the editing application.
+omitted    |Boolean       |If set to `true` fully omits this field in the content delivery API and preview API responses. The content management API is not affected by this.
```
Comment (realityking) [inline apiary/_partials/content-types-description.md]: APIs are not capitalised.

File: apiary/_partials/delivery-and-preview-resources.apib [hunk 14]
```diff
@@ -106,20 +106,20 @@ Entries are the documents contained within a space. They might represent blog po
...
 # Group Assets
 
-Assets represent files in a Space. An asset can be any kind of file: an image, a video, an audio file, a PDF or any other filetype. Assets are usually attached to Entries through Links.
+Assets represent files in a space. An asset can be any kind of file: an image, a video, an audio file, a PDF or any other filetype. Assets are usually attached to entries through links.
 
-Assets can optionally be localized by providing separate files for each locale. Those Assets which are not localized simply provide a single file under the default locale.
+Assets can optionally be localized by providing separate files for each locale. Those assets which are not localized simply provide a single file under the default locale.
 
-When querying the Content Delivery API for Entries which contain Links to Assets then all Assets will be included by default.
+When querying the content delivery API for entries which contain links to assets then all assets will be included by default.
```
Comment (realityking) [inline apiary/_partials/delivery-and-preview-resources.apib]: API not capitalised.

File: apiary/_partials/delivery-and-preview-resources.apib [hunk 15]
```diff
@@ -192,24 +192,24 @@ All of this, with examples, is documented in our [Images API](https://www.conten
...
-Learn more [on our Developer Center](https://www.contentful.com/developers/docs/concepts/links/).
+Learn more in the [developer center](https://www.contentful.com/developers/docs/concepts/links/).
 
 ## Retrieval of linked items [/spaces/{space_id}/entries?access_token={access_token}&include={value}]
 
-When you have related content (e.g. Entries with links to image Assets) it's possible to include both search results and related data in a single request. To do so, the `include` parameter should be used, specifying the number of levels to be resolved. The maximum number of inclusions is `10`.
+When you have related content (e.g. entries with links to image assets) it's possible to include both search results and related data in a single request. To do so, the `include` parameter should be used, specifying the number of levels to be resolved. The maximum number of inclusions is `10`.
 
-This feature is only available on the Delivery and Preview APIs.
+This feature is only available on the content delivery and preview API.
```
Comment (realityking) [inline apiary/_partials/delivery-and-preview-resources.apib]: API not capitalised.

File: apiary/_partials/delivery-and-preview-resources.apib [hunk 16]
```diff
@@ -489,20 +489,20 @@ Contentful works great with location-enabled content:
 - Show places within a specific map area
 - Find places in the vicinity of another place
 
-Entries can have location fields. For example, imagine a City Entry
+Entries can have location fields. For example, imagine an entry `city`
```
Comment (realityking) [inline apiary/_partials/delivery-and-preview-resources.apib]: Shouldn't that be a "content_type `city`"

File: apiary/cma.apib [hunk 17]
```diff
@@ -224,38 +224,38 @@ Finally, in these responses, you can also specify validations and determine whet
...
-`in`|Takes an array of values and validates that the field value is in this array.|Text, Symbol, Integer, Number| `{"in": ["General", "iOS", "Android"]}`
-`linkMimetypeGroup`|Takes a MimeType group name and validates that the link points to an asset of this group.|Links to Assets|`{"linkMimetypeGroup": "image"}`
-`size`|Takes optional min and max parameters and validates the size of the array (number of objects in it).|Arrays, Text, Symbol|`{"size": { "min": 5, "max": 20}}`
-`range`|Takes optional min and max parameters and validates the range of a value.|Number, Integer|`{"range": { "min": 5, "max": 20}}`
-`regexp`|Takes a string that reflects a JS regex and flags, validates against a string. See [JS Reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp) for the parameters.|Text, Symbol|`{"regexp": {"pattern": "^such", "flags": "im"}}`
+`linkContentType`|Takes an array of content type ids and validates that the link points to an entry of that content type.|Links to entries|`{"linkContentType": ["post","doc","product"]}`
```
Comment (realityking) [inline apiary/cma.apib]: Data types not capitalised in this table.

File: apiary/_partials/delivery-and-preview-resources.apib [hunk 18]
```diff
@@ -192,24 +192,24 @@ All of this, with examples, is documented in our [Images API](https://www.conten
...
-Learn more [on our Developer Center](https://www.contentful.com/developers/docs/concepts/links/).
+Learn more in the [developer center](https://www.contentful.com/developers/docs/concepts/links/).
 
 ## Retrieval of linked items [/spaces/{space_id}/entries?access_token={access_token}&include={value}]
 
-When you have related content (e.g. Entries with links to image Assets) it's possible to include both search results and related data in a single request. To do so, the `include` parameter should be used, specifying the number of levels to be resolved. The maximum number of inclusions is `10`.
+When you have related content (e.g. entries with links to image assets) it's possible to include both search results and related data in a single request. To do so, the `include` parameter should be used, specifying the number of levels to be resolved. The maximum number of inclusions is `10`.
 
-This feature is only available on the Delivery and Preview APIs.
+This feature is only available on the content delivery and Preview API.
```
Comment (realityking) [inline apiary/_partials/delivery-and-preview-resources.apib]: API not capitalised.

File: apiary/cma.apib [hunk 19]
```diff
@@ -839,29 +839,29 @@ cannot be recreated by creating the same locale again.
             Content-Type: application/vnd.contentful.management.v1+json
 + Response 204
 
-# Group Search Parameters
+# Group Search parameters
 
-[See CDA documentation](https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/search-parameters) for a list of search parameters supported for querying resources. Please note that the CMA does not support the `include` functionality.
+See the  [reference for the Content Delivery API](https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/search-parameters) for a list of search parameters supported for querying resources. Please note that the Content Management API does not support the `include` functionality.
```
Comment (realityking) [inline apiary/cma.apib]: Double (or triple?) space. Also the sentence sounds weird, I'd say "See the reference *documentation* of the Content Delivery API"

File: docs/concepts/sync.md [hunk 20]
```diff
@@ -15,22 +15,22 @@ Instead, using the Sync API, applications sync periodically. Depending on the us
...
-## The Upside
+## The upside
 
 Compared to the other strategies, syncing with delta updates has many advantages:
 
 - Use as little data as possible: Most mobile data plans have limits on how much data can be transferred (at full speed) within a given time period. By transferring only actual changes and not transferring everything every time, the amount of data of a synchronization is very small. Keeping the sync small also means improved chances of success when a mobile connection is slow.
 - Fast synchronization: Because individual syncs are small they will take much less time than repeatedly loading data that hasn't changed.
 - As little resource usage as possible: Less data to process also means less resources required for processing the data and less waiting time for users of mobile applications.
-- Entirety of content: the initial sync will transfer all items since the creation of the Space alongside all available localizations, which doesn't happen in the Delivery API.
+- Entirety of content: the initial sync will transfer all items since the creation of the Space alongside all available localizations, which doesn't happen in the delivery API.
```
Comment (realityking) [inline docs/concepts/sync.md]: Shouldn't this be "Content Delivery API"?

File: apiary/cma.apib (hunk 19 above)
Comment (hollensteiner) [inline apiary/cma.apib]: whitespace iz beautiful!

File: docs/php/tutorials/getting-started-with-contentful-and-symfony.md [hunk 21]
```diff
@@ -52,7 +52,7 @@ contentful:
     token: b4c0n73n7fu1
 ~~~
 
-This is the minimum configuration necessary to use ContentfulBundle. To use the Preview API instead of the Delivery API,
+This is the minimum configuration necessary to use ContentfulBundle. To use the Preview API instead of the delivery API,
```
Comment (realityking) [inline docs/php/tutorials/getting-started-with-contentful-and-symfony.md]: This (and the next change blow it looks weird because "Preview" is capitalised but "delivery" is not.

File: docs/references/http-details.md [hunk 22]
```diff
@@ -4,49 +4,35 @@ page: :docsHttpDetails
...
 
 [CORS (Cross-origin resource sharing)](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing)
-allows JavaScript web apps to make HTTP requests to other domains.
-This is very important for third party web apps using Contentful:
-Without CORS, a JS app on https://example.com couldn't access the
-Content Management or the Content Delivery API because they're on
-contentful.com which is a different domain.
-In short, without CORS it wouldn't be possible to easily access
-Contentful's APIs from a third party web app.
+allows JavaScript web apps to make HTTP requests to other domains. This is very important for third party web apps using Contentful: Without CORS, a JS app on https://example.com couldn't access the content management or the Content Delivery API because they're on contentful.com which is a different domain. In short, without CORS it wouldn't be possible to easily access Contentful's APIs from a third party web app.
```
Comment (realityking) [inline docs/references/http-details.md]: "content management" needs to be capitalised

File: docs/ruby/index.md [hunk 23]
```diff
@@ -61,7 +61,7 @@ A collection of useful things to help make it easier to integrate Contentful int
 Links to the source code of our in house built apps. Use them as examples when building your own product with Contentful.
 
 ### Rails Product Demo
-Simple Rails and Contentful Application.<br>
+Simple rails and Contentful application.<br>
```
Comment (realityking) [inline docs/ruby/index.md]: "Rails" needs to be capitalised

File: docs/ruby/tutorials/getting-started-with-contentful-and-ruby.md [hunk 24]
```diff
@@ -37,19 +37,19 @@ client = Contentful::Client.new(
...
 
-In Contentful, we separate content between Entries, which contain your data and relationships with other
-content or images, and Assets, which represent static content, like images, and are served as files. You can read more
-about that in our [Data Model Concepts guide][6].
+In Contentful, we separate content between entries, which contain your data and relationships with other
+content or images, and assets, which represent static content, like images, and are served as files. You can read more
+about that in our [data model concepts guide][6].
 
-In this section we'll address Entries. Assets will be addressed on a later section.
+In this section we'll address entries. assets will be addressed on a later section.
```
Comment (realityking) [inline docs/ruby/tutorials/getting-started-with-contentful-and-ruby.md]: "Assets" needs to be capitalised as it's at the beginning of a sentence

File: docs/ruby/tutorials/using-contentful-bootstrap-for-keeping-up-with-your-spaces.md [hunk 25]
```diff
@@ -4,16 +4,16 @@ page: :docsContentfulBootstrap
...
-* Setup your account from the Command Line
-* Create a Space
-* Create and use Templates for your Spaces and Entries
-* Using your Bootstrap generated Content
-* Generate Templates from existing Spaces
-* Generate API Access Tokens
+* Setup your account from the command line
+* Create a space
+* Create and use templates for your spaces and entries
+* Using your bootstrap generated Content
```
Comment (realityking) [inline docs/ruby/tutorials/using-contentful-bootstrap-for-keeping-up-with-your-spaces.md]: Why is "Content" capitalised?

File: docs/ruby/tutorials/using-contentful-bootstrap-for-keeping-up-with-your-spaces.md [hunk 26]
```diff
@@ -55,21 +55,21 @@ If you want to, and I leave this as an exercise to the reader, you can look for
 
 We will now proceed to explain some of their use cases.
 
-### Creating an Account via the Command Line
+### Creating an account via the command line
 
-Whenever you run `create_space` or `generate_token` the program will check if you already have Contentful Management API Credentials.
+Whenever you run `create_space` or `generate_token` the program will check if you already have contentful management API credentials.
```
Comment (realityking) [inline docs/ruby/tutorials/using-contentful-bootstrap-for-keeping-up-with-your-spaces.md]: "contentful management API" needs to be capitalised

File: docs/ruby/tutorials/using-contentful-bootstrap-for-keeping-up-with-your-spaces.md [hunk 27]
```diff
@@ -4,16 +4,16 @@ page: :docsContentfulBootstrap
...
-* Create and use Templates for your Spaces and Entries
-* Using your Bootstrap generated Content
-* Generate Templates from existing Spaces
-* Generate API Access Tokens
+* Setup your account from the command line
+* Create a space
+* Create and use templates for your spaces and entries
+* Using your bootstrap generated Content
+* Generate templates from existing spaces
+* Generate API access tokens
```
Comment (realityking) [inline docs/ruby/tutorials/using-contentful-bootstrap-for-keeping-up-with-your-spaces.md]: I think this is now an "API key" (this is wrong in the whole document)

File: docs/ruby/tutorials/using-contentful-bootstrap-for-keeping-up-with-your-spaces.md [hunk 28]
```diff
@@ -198,16 +198,16 @@ client.entries.first.my_text
 # => "some fancy text"
 ~~~
 
-With this, you can now use your `contentful_bootstrap` generated data. If you want to use [Contentful Management API][4] for managing your content, just make sure to use
-the Management Key as explained above.
+With this, you can now use your `contentful_bootstrap` generated data. If you want to use the [contentful management API][4] for managing your content, just make sure to use
```
Comment (realityking) [inline docs/ruby/tutorials/using-contentful-bootstrap-for-keeping-up-with-your-spaces.md]: "contentful management API" => "Content Management API"

File: docs/tutorials/general/anatomy-cda-request.md [hunk 29]
```diff
@@ -4,31 +4,31 @@ page: :docsAnatomyCDARequest
 
 ## Overview
 
-Contentful's Delivery API (CDA) is a read-only API for retrieving content from Contentful. All content, both JSON and binary, is fetched from the server closest to an user's location by using our global CDN.
+Contentful's Content Delivery API (CDA) is a read-only API for retrieving content from Contentful. All content, both JSON and binary, is fetched from the server closest to an user's location by using our global CDN.
 
 This article goes into detail about how the requests and responses work using the CDA. If you want to keep it simple, you can use the SDKs we're offering for the most popular [programming languages](/developers/docs/#libraries). But if you want to know exactly how the API works, this page is for you.
 
-To get started, for every request, clients [need to provide an access token](/developers/docs/references/authentication/), which is created per Space and used to delimit audiences and content classes.
+To get started, for every request, clients [need to provide an access token](/developers/docs/references/authentication/), which is created per space and used to delimit audiences and content classes.
```
Comment (realityking) [inline docs/tutorials/general/anatomy-cda-request.md]: again  access token vs API key
